### ISSUE TITLE
deafult ifl-processors added although cps are requested

### ISSUE THREAD

**Issue by scheuran** [zhmcclient/python-zhmcclient#258]
### Actual behavior

Start a partition
$ zhmc --debug  partition create P000S67B --name DPM_IMAGE_CREATE --description 'Setup used by automation (Andreas Scheuring) - Installing rhel' --cp-processors '1' --initial-memory '4096' --maximum-memory '4096' --processor-mode shared --boot-ftp-host '...' --boot-ftp-insfile 'foo' --boot-ftp-password '...' --boot-ftp-username '...' --log all=debug

The log output shows the following:

DEBUG zhmcclient.api: ==> PartitionManager.create(), args: (<zhmcclient._partition.PartitionManager object at 0x7f3eea316910>, {'boot-ftp-password': u'...', 'boot-ftp-insfile': u'ftpboot/dpm_image_install__rhel_7.3.ins', 'initial-memory': 4096, 'name': u'DPM_IMAGE_CREATE', 'cp-processors': 1, 'ifl-processors': 1, 'boot-ftp-username': u'...', 'boot-ftp-host': u'...', 'maximum-memory': 4096, 'processor-mode': 'shared', 'boot-device': 'ftp', 'description': u'Setup used by automation (Andreas Scheuring) - Installing rhel'})
...
Error: HTTPError: 400,15: Expected to find exactly one of the following fields in the document: 'ifl-processors' or 'cp-processors' but found more than one

-> IFL-processors got added

### Expected behavior
This command starts a partition with 1 CP.

### Execution environment

zhmc, version 0.12.1.dev11
* Operating system (type+version): ubuntu 14
* HMC version:
* CPC version:

**Comment by andy-maier** [comment]
After talking with Andreas S., here is the corresponding request to the HMC:
```
DEBUG zhmcclient.hmc: HMC request:
    POST https://...:6794/api/cpcs/fa1f2466-12df-311a-804c-4ed2cc1d6564/partitions,
    headers: {'X-API-Session': '...', 'Content-type': 'application/json', 'Accept': '*/*'},
    content(max.1000): '{"boot-ftp-password": "...", "boot-ftp-insfile": "ftpboot/dpm_image_install__rhel_7.3.ins",
        "initial-memory": 4096, "name": "DPM_IMAGE_CREATE", "cp-processors": 1,
        "ifl-processors": 1, "boot-ftp-username": "...", "boot-ftp-host": "...", "maximum-memory": 4096,
        "processor-mode": "shared", "boot-device": "ftp", "description": "Setup used by automation
        (Andreas Scheuring) - Installing rhel"}'
```

**Comment by andy-maier** [comment]
Fixed in PR #259.
